contributing_to_docs/doc_guidelines.adoc

```diff
@@ -85,6 +85,36 @@ Example:
 = Basic CLI commands
 ----
 
+== Text snippet file metadata
+Every text snippet should be placed in the snippets folder and should contain the following metadata at the top:
+
+----
+// Text snippet included in the following assemblies: <1>
+//
+// * list of assemblies where this text snippet is included
+// Text snippet included in the following modules:    <2>
```

Comment:
For readability, maybe include a blank line before this one to separate the lists.